app.py

```diff
@@ -18,11 +17,11 @@
 
 colorscale = cl.scales['9']['qual']['Paired']
 
-df_symbol = pd.read_csv('tickers.csv')
+df = pd.read_csv('stock-ticker.csv')
```

Comment:
@bcdunbar would it be possible to add the csv to github: https://github.com/plotly/datasets then reference the url
I think that makes the example a little easier for people to run